lib/keysim.js

```diff
@@ -189,24 +189,37 @@ export class Keyboard {
    * @param {Keystroke} keystroke
    * @param {HTMLElement} target
    * @param {boolean=} transitionModifiers
+   * @param {Object.<eventString, boolean>} events
    */
-  dispatchEventsForKeystroke(keystroke, target, transitionModifiers=true) {
+  dispatchEventsForKeystroke(keystroke, target, transitionModifiers=true, events) {
```

Comment:
I would prefer doing this as a bitmask instead of an object with keys and boolean values. Then we can have a default value of something like `events=KeyEvents.ALL` and checking for events like `if (events & KeyEvents.KEYDOWN) {`.